src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -396,10 +395,13 @@ private static ProjectDependency createConfigurationDependency(Project project,
     }
 
     /** Create a dependency to {@code toConfiguration}, where the latter should exist in the given {@code project}. */
-    private static Dependency createConfigurationDependencyWithScope(
-            Project project, Configuration toConfiguration, GcvScope scope) {
-        ModuleDependency dep = GradleWorkarounds.fixAttributesOfModuleDependency(
-                project.getObjects(), createConfigurationDependency(project, toConfiguration));
+    private static Dependency createConfigurationDependencyWithScope(Project project, GcvScope scope) {
+        ProjectDependency projectDependency = (ProjectDependency) project.getDependencies().create(project);
+        projectDependency.capabilities(moduleDependencyCapabilitiesHandler -> {
+            moduleDependencyCapabilitiesHandler.requireCapabilities(capabilityFor(project, scope));
+        });
+        ModuleDependency dep =
+                GradleWorkarounds.fixAttributesOfModuleDependency(project.getObjects(), projectDependency);
```

Comment:
actually this line is no longer necessary either, I think you can just configure the ProjectDependency directly

Reply:
Not sure what this means? Can I just remove the line/workaround entirely now (won't it still break for people on <5.6?)?

Reply:
Seems to pass the tests on 5.3 without this line